war/src/main/js/pluginSetupWizardGui.js

```diff
@@ -1282,6 +1282,20 @@ var createPluginSetupWizard = function (appendTarget) {
   // do this so the page isn't blank while doing connectivity checks and other downloads
   setPanel(loadingPanel);
 
+  /* exported defaultUpdateSiteId */
```

Comment:
There's an error now: 
![image](https://github.com/jenkinsci/jenkins/assets/91559310/533bcfb3-9154-4947-a4fb-9d468e200774)

Reply:
@yaroslavafenkin Can you let me know how did you reproduced this error? So I can also test it

Reply:
Start a fresh instance of Jenkins, it would be the first thing you'd see after entering initial admin password.